rhp/v3/rhp.go

```diff
@@ -71,12 +72,16 @@ func PayByContract(rev *types.FileContractRevision, amount types.Currency, refun
 	for i, o := range rev.NewMissedProofOutputs {
 		newMissed[i] = o.Value
 	}
+	ra := modules.ZeroAccountID
```

Comment:
Can we drop support for the `ZeroAccount`? It looks like it's only used in `RPCFundAccount` and has little benefit.